MediaBrowser.Model/Plugins/PluginStatus.cs

```diff
@@ -36,7 +36,7 @@ public enum PluginStatus
         /// <summary>
         /// This plugin has been superceded by another version.
```

Comment:
```suggestion
        /// This plugin has been superseded by another version.
```

Reply:
Hmm, I wonder how I missed this (my guess is that it was a case thing -- it looks like my other corrections were mostly `Initial` instead of `lowercase`). I can merge this into the current commit for that fix, or I can just commit it using github's ui and let it have a signed-off-by/co-authored attribute.

(I can't tell if this repository or you care about credit.)

Anyway, it's queued -- lemme know if you have a preference.

Also whether this repository is ok w/ force-pushes or prefers append-only. (It looks like it uses squash-merge, so it probably doesn't matter much...)

Reply:
No thank I don't care about co-author commit no problem